presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -269,4 +271,16 @@ public FeaturesConfig setRegexLibrary(RegexLibrary regexLibrary)
         this.regexLibrary = regexLibrary;
         return this;
     }
+
+    public boolean isCharPadSpaces()
+    {
+        return charPadSpaces;
+    }
+
+    @Config("char.pad-spaces")
```

Comment:
Can you add a description of what this config does? It looks like `true` means implicitly pad all `char(x)` with spaces on the right, whereas `false` effectively treats them as `varchar(x)`